lib/bolt_ext/server.rb

```diff
@@ -71,3 +77,19 @@ class TransportAPI < Sinatra::Base
     [500, "500: Unknown error: #{e.message}"]
   end
 end
+
+class ValidateRequest
+  def initialize(endpoint, request)
+    case endpoint
```

Comment:
It might be easier to just have the schema uri be from the endpoint that's passed in, like:
`@schema = "lib/bolt_ext/schemas/#{endpoint}.json"`

Then you don't need the case statement.

Reply:
Actually, this seems like it probably doesn't need to be a whole class? I think having:
`schema_error = JSON::Validator.fully_validate("lib/bolt_ext/schemas/ssh-run-task.json", body)` would work.